ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,338 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
```

Comment:
Imports are grouped differently elsewhere.

1. futures
2. standard lib
3. third party
4. local imports

See frontend.py as a good example. This grouping helps us know where things come from.